projects/graph/graph.py

```diff
@@ -1,48 +1,115 @@
 """
 Simple graph implementation
 """
+
 from util import Stack, Queue  # These may come in handy
 
+
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
+
     def __init__(self):
         self.vertices = {}
+
     def add_vertex(self, vertex):
         """
         Add a vertex to the graph.
         """
+        self.vertices[vertex] = set()
```

Comment:
I would add conditioning here to prevent vertices that already exist in the graph from being recreated again.